doc/user/builders-writing.xml

```diff
@@ -496,13 +496,15 @@ def build_function(target, source, env):
 
     <para>
 
-    The builder function must
-    return a <literal>0</literal> or <literal>None</literal> value
-    if the target(s) are built successfully.
-    The builder function
-    may raise an exception
-    or return any non-zero value
-    to indicate that the build is unsuccessful.
+    The function will be constructed as a SCons FunctionAction and
+    must return a <literal>0</literal> or <literal>None</literal>
+    value if the target(s) are built successfully.  The function may
+    raise an exception or return any non-zero value to indicate that
+    the build is unsuccessful.
+
+    For more information on Actions see the <ulink
+    url="https://scons.org/doc/production/HTML/scons-man.html#action_objects"
```

Comment:
I don't think we have any other URL's in the users guide. I'd prefer something like 'refer to the section on Action Objects in the man page', and/or just copying that content here. But probably just the refer...

Reply:
I will make this change, though I think the UX of having a link is much better, I understand maintaining consistency.